Core/clim-core/panes.lisp

```diff
@@ -363,11 +363,38 @@ order to produce a double-click")
   (print-unreadable-object (pane sink :type t :identity t)
     (prin1 (pane-name pane) sink)))
 
+(defgeneric find-concrete-pane-class (pane-realizer pane-type)
+  (:documentation "Resolves abstract pane name to a concrete pane ~
+class which may be instatinated. Function specializes on its first ~
+argument. Returns NIL if no concrete class exists.")
+  (:method ((realizer t) pane-type)
+    ;; Default method tries to resolve the pane name as specified by a
+    ;; convention mentioned in the spec. Function is little
+    ;; complicated because we preserve old semantics adding rules to
+    ;; the class name resolution. Resolution works as follows:
+    ;;
+    ;; 1. Abstract mapping always takes a priority. When it exists we
+    ;;    expect that class is defined and signal error otherwise.
+    ;; 2. When the symbol is in clim/climi/keyword package:
+    ;;    - look for a class `CLIMI::{symbol-name}-PANE'
+    ;;    - look for a class `CLIMI::{symbol-name}'
```

Comment:
The emacs notation usually uses lower case for symbols and uppercase for meta-syntactic variables:

```
; `climi::SYMBOL-NAME-pane'
```

I think.

Reply:
in that case: are SYMBOL and NAME separate arguments or a single argument? is it possible to distinguish with this notation?

Reply:
I've changed it to

````
    ;;    - look for a class `climi::{SYMBOL-NAME}-pane'
    ;;    - look for a class `climi::{SYMBOL-NAME}'
```

please resolve if you agree.

Reply:
That's fine given the ambiguity you pointed out.